int32 auc_id
int32 robot_id